AGENTS.md

```
@@ -1,71 +1,133 @@
+  - `cat-launcher/src/pages/{feature_name}/hooks`: Hooks specific to this feature.
+  - `cat-launcher/src/pages/{feature_name}/lib`: Utilities specific to this feature.
+  - `cat-launcher/src/pages/{feature_name}/store`: Redux store specific to this feature.
+  
+Always follow the directory structure above when creating a new feature.
 
-### Frontend (TypeScript/React)
+# Backend
 
--   **Data Fetching:** `@tanstack/react-query` is used for all asynchronous operations and state caching.
-    -   Query keys are centrally managed in `src/lib/queryKeys.ts`. Using string literals or "magic literals" for query keys is not allowed; always import them from the central file.
+* Tools: `tauri`, `thiserror`, `tokio`, 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)